GAE/src/com/gallatinsystems/surveyal/dao/SurveyedLocaleDao.java

```diff
@@ -296,6 +296,8 @@ public SurveyedLocaleDao() {
             appendNonNullParam("displayName", filterString, paramString,
                     "String", endString, paramMap, LT_OP);
             query.setOrdering("displayName asc"); //necessary if inequality ops present
+            //Alternative custom filterString:
+            // surveyGroupId == surveyGroupIdParam0 && displayName.startsWith(displayNameParam1)
```

Comment:
it's commented out, will this work?

Reply:
Yes, this shows an alternative implementation. Something that would also work.